Release/bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceMenuMod/init.lua

```diff
@@ -2340,15 +2345,16 @@ function AMM:SetupCustomEntities()
 						end
 					end
 
-					-- Setup TweakDB Records
-					if entity.record ~= nil then
-						TweakDB:CloneRecord(entity_path, entity.record)
-					else
-						TweakDB:CloneRecord(entity_path, "Character.CitizenRichFemaleCasual")
-					end
-
-    				TweakDB:SetFlat(entity_path..".entityTemplatePath", entity.path)
+					if not TweakDB:GetRecord(entity_path) then
+						-- Setup TweakDB Records
+						if entity.record ~= nil then
```

Comment:
Is the issue here that only the `entity.record` link is broken even though the record exists, but it was assumed that the link missing meant the record was missing too?

Reply:
My theory is that maybe entity saving was made persistent at some point, because the loop runs every time on startup and spams one line in the log file for every entity that it fails to register